src/components/ChatLog.jsx

```diff
@@ -0,0 +1,34 @@
+import PropTypes from 'prop-types';
+import ChatEntry from './ChatEntry';
+import './ChatLog.css';
+
+function ChatLog({ entries, onLikeToggle }) {
+  return (
+    <section className="chat-log">
+      {entries.map((entry) => (
+        <ChatEntry
+          key={entry.id}
+          {...entry}
+          onLikeToggle={() => onLikeToggle(entry.id)}
```

Comment:
We actually can just pass the function like: 
```suggestion
          onLikeToggle={onLikeToggle}
```